.github/workflows/tests.yml

```diff
@@ -13,11 +13,15 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [7.3, 7.4, '8.0', 8.1]
```

Comment:
Best to leave this be because otherwise It'll show up as PHP 8 while you want to specifically note PHP 8.0

Reply:
Yeah, fair point!